<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataInputSplitLinePage</class>
 <widget class="QWidget" name="DataInputSplitLinePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}
QLabel
{
	border:1px solid lightgray;
    color:white;
font:16px
}

QPushButton
{
	font:16px;
    color: rgb(240,240,240);
	background-color: rgb(0, 0, 0);
	border-top: 2px solid white;	
	border-left: 2px solid white;
}

QPushButton::disabled
{
	font:16px;
	color: rgb(147, 148, 148);
	background-color: black;
	border-bottom: 2px solid white;	
	border-right: 2px solid white;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>371</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:27px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNormalLine">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;</string>
   </property>
   <property name="text">
    <string>常规线路</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFreightLine">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;</string>
   </property>
   <property name="text">
    <string>货运线路</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>161</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
font:17px;</string>
   </property>
   <property name="text">
    <string>说明：
常规线路：预告点与
强迫断点之间距离为
170cm；
货运专线：预告点与
强迫断点之间距离为
265cm；</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>26</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:20px;
 border:0px;</string>
   </property>
   <property name="text">
    <string>分相路线返回模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMode">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
